common/lib/xmodule/xmodule/x_module.py

```diff
@@ -39,6 +39,7 @@
 
 XMODULE_METRIC_NAME = 'edxapp.xmodule'
 XMODULE_DURATION_METRIC_NAME = XMODULE_METRIC_NAME + '.duration'
+XMODULE_METRIC_SAMPLE_RATE = 0.1
```

Comment:
How likely is it that this 10% number is correct? How likely is it that it'll need to be changed in a manner more quickly than an edx-platform release? (If your answers are "Quite." and "Not at all.", then fine.)